vowpalwabbit/example_predict.h

```diff
@@ -47,6 +49,7 @@ struct example_predict
   // Interactions are specified by this vector of vectors of unsigned characters, where each vector is an interaction
   // and each char is a namespace.
   std::vector<std::vector<namespace_index>>* interactions;
+  predict_data _predict_data;
```

Comment:
```suggestion
  predict_data predict_data;
```
this is public and intended to be accessed from outside of the class, so var name should not be prepended with underscore